CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Changed
 * **css** `dso-icons.svg`: Inline `<style>` elementen met `color: <kleur>` vervangen met `fill="<kleur>"` tbv. Content-security-policy issue met SVG spritesheet voor iconen ([#1600](https://github.com/dso-toolkit/dso-toolkit/issues/1600))
+* **css + sources:** Form Group: Search Bar naar @dso-toolkit/css: ([#1614](https://github.com/dso-toolkit/dso-toolkit/issues/1614))
```

Comment:
Deze graag markeren als BREAKING, op mijn verzoek hebben we de markup gewijzigd.